src/openforms/contrib/kvk/client.py

```diff
@@ -29,16 +29,18 @@ class SearchParams(TypedDict, total=False):
     kvkNummer: str
     rsin: str
     vestigingsnummer: str
-    handelsnaam: str
+    naam: str
     straatnaam: str
     plaats: str
     postcode: str
-    huisnummer: str
+    huisnummer: int
+    huisletter: str
+    postbusnummer: int
     huisnummerToevoeging: str
-    type: str
-    InclusiefInactieveRegistraties: Literal["true", "false"]
+    type: list[str]
+    inclusiefInactieveRegistraties: Literal["true", "false"]
     pagina: int  # [1, 1000]
-    aantal: int  # [1, 100]
+    resultatenPerPagina: int  # [1, 100] - default is 10
```

Comment:
I guess if tests are passing it means we weren't making use of these fields?

Reply:
Yes, I understood that we declare here all the possible query params, but in fact we only validate against `kvkNummer`, `rsin` and `vestigingsnummer`